USAGE.md

```diff
@@ -0,0 +1,134 @@
+Running Ansibullbot
+===================
+
+How to deploy and run ansibullbot for one or more collections.
+
+Requirements
+------------
+
+* AWS account
+* Github account
+* Python2.7 environment
+
+Setup
+-----
+
+* Clone this repo (with the correct branch) to your local machine.
```

Comment:
Is "correct branch" needed now we have a dedicated repo and use the default `main` branch?

Reply:
Probably not, good catch